### PR TITLE
the naturality conditions for ann and distrib were missing.

### PR DESCRIPTION
**Pull request by JacquesCarette**
So this adds them. And a partial refactoring to use variables.

### CODE REVIEW

**Comment by JacquesCarette** [comment]
(Asking multiple people for review because this change is blocking me on other work that I'm doing, so I'd like to have this merged sooner rather than later)

File: src/Categories/Category/RigCategory.agda [hunk 1]
```diff
@@ -17,15 +17,22 @@ open import Categories.NaturalTransformation.NaturalIsomorphism
...
+
+private
+  variable
+    X Y Z W A B : Obj
+    f g h i a b : X ⇒ Y
+
 -- Should probably split out Distributive Category out and make this be 'over' that.
 record RigCategory {M⊎ M× : Monoidal C} (S⊎ : Symmetric M⊎)
-   (S× : Symmetric M×) : Set (o ⊔ ℓ ⊔ e) where
+   (S× : Symmetric M×) : Set (o ⊔ ℓ ⊔ e) where
```
Comment (Taneb) [inline src/Categories/Category/RigCategory.agda:32]: Whitespace at end of line?

File: src/Categories/Category/RigCategory.agda [hunk 2]
```diff
@@ -72,6 +79,13 @@ record RigCategory {M⊎ M× : Monoidal C} (S⊎ : Symmetric M⊎)
     ⊕α⇐ = M⊎.associator.to
 
 
+  -- that ann and distrib are natural was missing
```
Comment (Taneb) [inline src/Categories/Category/RigCategory.agda:82]: I don't think this comment should be in the code. Maybe change to something like  `naturality conditions for ann and distrib`

**Review by Taneb — CHANGES_REQUESTED** [review]

File: src/Categories/Category/RigCategory.agda (hunk 1 above)
Comment (JacquesCarette) [reply src/Categories/Category/RigCategory.agda]: Fixed

File: src/Categories/Category/RigCategory.agda (hunk 2 above)
Comment (JacquesCarette) [reply src/Categories/Category/RigCategory.agda]: Good point - fixed.

**Review by Taneb — APPROVED** [review]

**Pull request merged**